packages/typescript/src/generated.ts

```diff
@@ -163,6 +163,7 @@ export type ConnectOAuthErrorPayload = {
   type: Type.ConnectOAuthError,
   user_guid: string,
   session_guid: string,
+  member_guid?: string,
```

Comment:
Even the type is flagged as optional. This is nice because it lets any other TS users know when they're using the field and expecting it to always have a value. Here's an example of an error message that you'd see if you used `member_guid` as a non-optional field:

```
    test/post_message_dispatch_test.ts:324:16 - error TS2532: Object is possibly 'undefined'.

    324               (payload.member_guid.valueOf())
                       ~~~~~~~~~~~~~~~~~~~
```